src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -209,6 +211,21 @@ export function ConfigEditor(props: Props) {
         label={selectors.components.ConfigEditor.Database.input}
         data-testid={selectors.components.ConfigEditor.Database.testID}
       />
+      <InlineField label="Send events to Amazon EventBridge" labelWidth={28}>
```

Comment:
If you add `style={{ alignItems: 'center' }}` the switch will be vertically aligned

Reply:
What is the easiest a way to view how the UI will look like in local machine? I tried `yarn dev` and `yarn watch` but they seem to be focussed on unit tests

Reply:
I use `yarn watch`, that shouldn't run the tests. With more details, what I run in development is:

 - `make run` in the main grafana/grafana repository
 - `yarn start` in grafana/grafana
 - `grafana/grafana/data/plugins/redshift-datasource` is a symbolic link pointing to my Redshift dist folder (`grafana/redshift-datasource/dist`)
 - `yarn watch` in grafana/redshift-datasource

Reply:
@andresmgot What do you think of adding the above steps to [CONTRIBUTING.md](https://github.com/grafana/redshift-datasource/blob/main/CONTRIBUTING.md)? Would be useful for people who are new to grafana plugin development like me :-)

Reply:
that's a good point. Let me check if that's already written somewhere and if not I will add it there